packages/grid/_modules_/grid/models/colDef/gridNumericOperators.ts

```diff
@@ -2,6 +2,14 @@ import { GridFilterInputValue } from '../../components/panel/filterPanel/GridFil
 import { GridFilterItem } from '../gridFilterItem';
 import { GridFilterOperator } from '../gridFilterOperator';
 
+const parseNumericValue = (value: string | number | null) => {
+  if (value == null) {
```

Comment:
For my own learning: Are you using `==` to catch `null` and `undefined`?

Reply:
Yes, and I tested both on the test file.
It's to avoid the `Number(null) === 0`